.github/workflows/fake-bpy-ci.yml

```
@@ -0,0 +1,84 @@
+name: fake-bpy-module CI
+
+on:
+  push:
+    branches:
+      # Temporarily trigger only on "ci_testing" branch
+      - ci_testing
+  # # Trigger on all PRs. By default, only run if activity type is opened, synchronize, or reopened
+  # pull_request:
+
+jobs:
+  build_modules:
+    name: Build fake-bpy modules
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+
+      - name: Blender downloads cache
+        uses: actions/cache@v2
+        id: blender-bin-cache
+        with:
+          path: blender-bin
+          key: blender-binaries-${{ runner.os }}-${{ hashFiles('**/tools/utils/download_blender.sh') }}
+
+      - name: Install python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install wheel sphinx
+          pip install -r requirements.txt
+
+      - name: Install Blender binary dependencies
+        run: sudo apt-get install -y libglu1-mesa
+
+      - name: Install fake-bpy-module tools
+        run: sudo apt-get install -y pandoc
+
+      - name: Download Blender files
+        if: steps.blender-bin-cache.outputs.cache-hit != 'true'
+        run: bash tools/utils/download_blender.sh all blender-bin
+
+        # alternatively to the github checkout, you can also use git.blender.org. See below.
+      - name: Checkout Blender from Github
+        uses: actions/checkout@v2
+        with:
+          repository: "blender/blender"
+          path: blender
+          fetch-depth: 0
+
+      # - name: Checkout Blender from git.blender.org
+      #   run: rm -rf ./blender && git clone "https://git.blender.org/blender.git" ./blender
+
+      - name: Set release version environment variable
+        run: |
+          # echo ::set-env name=RELEASE_VERSION::$(date +%Y%m%d).dev$(date +%H%M%S)
+          echo ::set-env name=RELEASE_VERSION::$(date +%Y%m%d)
+
+        # In the future, the following commands should be replaced by a matrix:
+        # https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix
```

It would be nicer to add `TODO:` tag.

```suggestion
        # TODO: In the future, the following commands should be replaced by a matrix:
        # https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix
```

---

added